Automattic/wp-calypso
# Add site profiler site metrics API hook

Related to https://github.com/Automattic/wp-calypso/issues/88971

## Proposed Changes

* Add site profiler metrics API hook to be used in the Migration upgrade plan step.

## Testing Instructions



* Checkout branch or use live link to verify Calypso isn't broken
* You can also optionally checkout the branch and test out the hook in a random component.

## Pre-merge Checklist
